.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 **/*.rs.bk
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+*.swp
```

Comment:
What kind of file is this? If it's created by some IDE it's better suited for `.git/info/exclude`.

Reply:
Created by vim. I'll move it to .git/info/exclude